src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1

```diff
@@ -93,7 +93,7 @@ function Compress-Archive
             $destinationParentDir = '.'
         }
 
-        $achiveFileName = [system.IO.Path]::GetFileName($DestinationPath)
+        $archiveFileName = [system.IO.Path]::GetFileName($DestinationPath)

```

Comment:
Lets not touch Archive module for a moment. We have a separate repo for it https://github.com/PowerShell/Microsoft.PowerShell.Archive and will probably replace archive code in this repo by a git submodule or something similar.